### PR TITLE
 [MIG] hr_employee_data_from_work_address to v10

### PR DESCRIPTION
**Pull request by hbrunn**

### CODE REVIEW

File: hr_employee_data_from_work_address/__manifest__.py [hunk 1]
```diff
@@ -1,46 +1,22 @@
...
+# Copyright 2015 Therp BV <http://therp.nl>
 {
     "name": "User and partner data from employee",
-    "version": "8.0.1.0.0",
-    "author": "Therp BV",
+    "version": "10.0.1.0.0",
+    "author": "Odoo Community Association (OCA),Therp BV",
     "license": "AGPL-3",
     "category": "Human Resources",
     "summary": "Update user's and partner's data fields from employee record",
```
Comment (tbaden) [inline hr_employee_data_from_work_address/__manifest__.py:10]: please website tag with value: https://github.com/OCA/hr

**Review by tbaden — CHANGES_REQUESTED** [review]
thanks for the work 👍 
i did a code review. lgtm, just a minor detail. please add website tag to the manifest

File: hr_employee_data_from_work_address/__manifest__.py (hunk 1 above)
Comment (hbrunn) [reply hr_employee_data_from_work_address/__manifest__.py:10]: thanks, done

**Review by tbaden — APPROVED** [review]

**Review by yucer — APPROVED** [review]
Good work.

Just one little doubt, but if was not related with these changes.

I did ask it under #602

File: hr_employee_data_from_work_address/__init__.py [hunk 2]
```diff
@@ -1,22 +1,4 @@
...
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
+# Copyright 2015 Therp BV <http://therp.nl>
 from . import models
 from .init_hook import pre_init_hook, post_init_hook, uninstall_hook
```
Comment (DucTruongKomit) [inline hr_employee_data_from_work_address/__init__.py:4]: I think it should be: `post_init_hook, pre_init_hook, uninstall_hook`

File: hr_employee_data_from_work_address/init_hook.py [hunk 3]
```diff
@@ -80,21 +55,50 @@ def adjust_employee_partners_post(env):
...
-        'select work_phone, work_email, mobile_phone, image '
+        'select work_phone, work_email, mobile_phone '
         'from hr_employee where id in %s order by id',
         (tuple(employees.ids),))
     employee_db_data = env.cr.dictfetchall()
     for employee, db_data in zip(employees, employee_db_data):
         employee.address_id = env['res.partner'].create({
             'employee': True,
-            'name': employee.name,
+            'name': employee.name or employee.display_name,
```
Comment (DucTruongKomit) [inline hr_employee_data_from_work_address/init_hook.py:65]: I think name of 'res.partner' should be updated by name employee, not display_name

**Review by DucTruongKomit — COMMENTED** [review]
I'm just sure about the order

**Pull request merged**